util/tls/tls.go

```diff
@@ -7,18 +7,18 @@ import (
 	"os"
 )
 
-type config struct {
+type Config struct {
```

Comment:
I needed it in the unit test here, couldn't find a way to do it without making it public:
https://github.com/stripe/veneur/pull/1029/files#diff-8d5bed61a3c0b7b0bfdd65dd4c93e647568b1a402840f5e302c0c922f6c58ea3R38

Reply:
Yeah the need for `type Config struct` seems redundant if the fields are exposed anyways. Could prolly just use `type Tls struct`

Reply:
> Could prolly just use type Tls struct
Doing this causes infinite recursion when unmarshalling yaml, because then the 
```
err := unmarshal(&config.Config)
```
will call 
```
func (config *Tls) UnmarshalYAML(unmarshal func(interface{}) error) error {
```
Type passed into `unmarshal` must be different than the receiver type to `UnmarshalYAML`.